fitlins/interfaces/bids.py

```diff
@@ -231,7 +231,7 @@ def _load_level1(self, runtime, analysis):
             TR = ents.pop('RepetitionTime', None)  # Required field in seconds
             if TR is None:  # But is unreliable (for now?)
                 preproc_files = analysis.layout.get(
-                    extension=['.nii', '.nii.gz'], **ents)
+                    extension=['.nii', '.nii.gz'], desc='preproc', **ents)
```

Comment:
Is there any chance `ents` could already contain `'desc'`? If so, this is setting us up for an error.

Reply:
It doesn't